application/src/main/java/com/sanction/lightning/facebook/FacebookProvider.java

```diff
@@ -27,4 +53,61 @@ public FacebookUser getFacebookUser() {
     return client.fetchObject("me", FacebookUser.class, Parameter.with("fields",
             "first_name, last_name, middle_name, gender, name, verified"));
   }
+
+  /**
+   * Gets the users photos.
+   * @return an array of URLs representing the users photos
+   */
+  public List<String> getFacebookUserPhotos() {
+    JsonObject photos = client.fetchObject("me/photos", JsonObject.class,
+            Parameter.with("type", "uploaded"), Parameter.with("fields", "link"));
+    JsonArray photosArray = photos.getJsonArray("data");
+
+    List<String> urlList = Lists.newArrayList();
+    for (int i = 0; i < photos.length(); i++) {
+      urlList.add(photosArray.getJsonObject(i).getString("link"));
+    }
+
+    return urlList;
+  }
+
+  /**
+   * Gets the users videos from facebook.
+   * @return list of url's representing the videos
+   */
+  public List<String> getFacebookUserVideos() {
+    JsonObject videos = client.fetchObject("me/videos", JsonObject.class,
+            Parameter.with("type","uploaded"), Parameter.with("fields","id, title"));
```

Comment:
For the `Parameter.with()` methods, there should be a space between the comma and the next argument.